### ISSUE TITLE
Sidebar won't open while on Facebook

### ISSUE THREAD

**Issue by Varlin** [Kiuryy/Bookmark_Sidebar#129]
Hi, everything is in the title. Like on the native 'new tab' page or in the chrome store, the sidebar seems incompatible with something on Facebook.

**Comment by Kiuryy** [comment]
The sidebar should work on facebook as well. I just tested it myself and could not reproduce any problems on the Newsfeed (https://www.facebook.com/), as well on some profile pages, I clicked through.

Maybe the extension got updated in the background and something not wiring up correctly. This normally should resolve after a browser restart. 

If you can provide more information about your extension configuration and the pages the sidebar is not working, this may help. The best way of doing so is simply writting a message using the feedback form of the extension, since there, many diagnosis data, as well as configuration infos are supplied to me, helping to reproducing and fix the issue.

If the problem no longer persists after a browser restart, please let me know :)

**Comment by Kiuryy** [comment]
I received your mail and took a look at this. From your explanation, the sidebar is working, but the hover area is not. I suppose the problem here is, that you may have zoomed in on facebook.com (e.g. 125%)?  If you zoom a website, there might be integer rounding happening when the browser calculates the pixel position of the mouse. The most left outer pixel then may not be recognized as first pixel and therefore not triggering the sidebar. 

In order to make the sidebar work with a zoomed website, you can increase the tolerance range in which the sidebar shows up to 2px (instead of 1px). You can find the according options in the settings of the extension under "Sidebar" > "Toggle" in the first box by clicking on "Configure area". With the latest version 1.20.0, rolling out this week, I changed the default width of the hover area to 2px, which should circumvent any zoom related problems.

If this is not the problem, please let me know, so I can investigate this further.

**Comment by Varlin** [comment]
Ow, you're right : it seems I had a zoom factor. I set it back to 100% and
it is now working perfectly. I didn't notice I had this zoom. Sorry to have
bothered you for nothing...

And again, thanks for this extension : this is thanks to this extension
that I finally decided to move from FF to Chrome. I definitively needed my
bookmarks sidebar !

Le mar. 16 mars 2021 à 21:51, Philipp König ***@***.***> a
écrit :
